import "DseSingleSpiritInfo.proto";

message DseAllSpiritInfo
{
	repeated DseSingleSpiritInfo		soldiers = 1;
}